token/program-2022/src/pod.rs

```diff
@@ -0,0 +1,99 @@
+//! Rewrites of the base state types represented as Pods
+
+use {
+    crate::{instruction::MAX_SIGNERS, state::PackedSizeOf},
+    bytemuck::{Pod, Zeroable},
+    solana_program::{program_pack::IsInitialized, pubkey::Pubkey},
+    spl_pod::primitives::PodU64,
+};
+
+/// Mint data stored as a Pod type
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq, Pod, Zeroable)]
+pub struct PodMint {
+    /// Optional authority used to mint new tokens. The mint authority may only
+    /// be provided during mint creation. If no mint authority is present
+    /// then the mint has a fixed supply and no further tokens may be
+    /// minted.
+    pub mint_authority: PodCOption<Pubkey>,
+    /// Total supply of tokens.
+    pub supply: PodU64,
+    /// Number of base 10 digits to the right of the decimal place.
+    pub decimals: u8,
+    /// Is not 0 if this structure has been initialized
+    pub is_initialized: u8,
+    /// Optional authority to freeze token accounts.
+    pub freeze_authority: PodCOption<Pubkey>,
+}
+impl IsInitialized for PodMint {
+    fn is_initialized(&self) -> bool {
+        self.is_initialized != 0
+    }
+}
+impl PackedSizeOf for PodMint {
+    const SIZE_OF: usize = std::mem::size_of::<Self>();
```

Comment:
While this expression is simple enough, maybe use `spl_pod::pod_get_packed_len::<Self>()` for consistency? Don't feel too strongly, though.

Reply:
That's a good point, will do!